ddtrace/internal/compat.py

```diff
@@ -226,3 +226,19 @@ def get_connection_response(
     CONTEXTVARS_IS_AVAILABLE = False
 else:
     CONTEXTVARS_IS_AVAILABLE = True
+
+
+try:
```

Comment:
👍🏻 I was going to suggest just moving the version info check up here rather than try/except, but actually this seems better since this isn't critical, if there is ever an install error and we don't have `pep562` then it shouldn't really matter much. Might be a pain for those trying to check for deprecations/migrate, but isn't mission critical.